test.sh

```diff
@@ -23,18 +23,16 @@ set -o xtrace
 # This script should work no matter where you call it from.
 cd "$(dirname "$0")"
 
-# Controller unit tests on native.
-pio test -e native
-# Make sure controller builds for target platform.
-pio run
-
 # Code style / bug-prone pattern checks (eg. clang-tidy)
 # WARNING: This might sometimes give different results for different people,
 # and different results on CI:
 # See https://community.platformio.org/t/no-version-of-tool-clangtidy-works-on-all-os/13219
 # Feel free to edit .clang_tidy to blacklist problematic checks.
-#
-# Disabled due to failures in nanoproto-generated code,
-# https://github.com/RespiraWorks/VentilatorSoftware/issues/146
-# pio check --pattern="*" --fail-on-defect=high
-# pio check -e native --pattern="*" --fail-on-defect=high
+pio check --fail-on-defect=high
+pio check -e native --fail-on-defect=high
+
+# Controller unit tests on native.
```

Comment:
What do you think about putting clang-tidy last, so that in the event you have a clang-tidy error, you still get test results on Travis?

(Ideally we'd get results from each of them independently, but that may be a bit much for our bash-fu.)

Reply:
Done, put it last. I also, after wasting more time than I'd like to admit, had to disable it in env:uno, because it keeps stumbling on its toolchain headers which are intended to be compiled with gcc.